### ISSUE TITLE
"welcome to wavebox" dashboard text is editable

### ISSUE THREAD

**Issue by TheSpyder** [wavebox/waveboxapp#1216]
Running Version 10.0.82.2 stable

![something has changed](https://user-images.githubusercontent.com/298292/74887523-ddeabd00-53c6-11ea-9c11-e134f8ca4195.png)

**Comment by nmat** [comment]
Yes, I assume this is actually intended in a way. This way you can modify the dashboard in any way you want.

**Comment by TheSpyder** [comment]
Certainly it's just the description field of a normal collection, but it still seems weird to customise this. I'm not even sure what collections are intended for, really.

**Comment by nmat** [comment]
The dashboard and collections is a way to create overviews of the whole wavebox workspace. In this way you can add your own collection of a things you want to quickly keep track of instead of having to go to each service one by one to check them out.

**Comment by TheSpyder** [comment]
But it doesn't do that? All I can seem to add to collections is links and app buttons, which don't do anything more than the sidebar buttons. The closest I get to tracking an app's status is the unread/unified list on the right. The whole feature seems pointless (or perhaps unfinished).
